draft/2022-07-27-this-week-in-rust.md

```diff
@@ -153,23 +141,25 @@ Rusty Events between 2022-07-27 - 2022-08-24 🦀
 
 ### North America
 
-* 2022-07-26 | Austin, TX, US | [ATX Rustaceans](https://www.meetup.com/atx-rustaceans/)
-    * [**First Official Meetup - July 2022**](https://www.meetup.com/atx-rustaceans/events/285878081/)
 * 2022-07-27 | Austin, TX, US | [Rust ATX](https://www.meetup.com/rust-atx/)
     * [**Rust Lunch**](https://www.meetup.com/rust-atx/events/287009519/)
-* 2022-08-05 | Portland, OR, US | [RustConf](https://rustconf.com/)
+* 2022-08-05 | Portland, OR, US + Virtual | [RustConf](https://rustconf.com/)
     * [**RustConf 2022**](https://rustconf.com/)
 * 2022-08-10 | Atlanta, GA, US | [Rust Atlanta](https://www.meetup.com/rust-atl/)
     * [**Grab a beer with fellow Rustaceans**](https://www.meetup.com/rust-atl/events/pczdssydclbnb/)
 * 2022-08-11 | Columbus, IL | [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
     * [**Monthly Meeting**](https://www.meetup.com/columbus-rs/events/dpkhgrydclbpb/)
 * 2022-08-16 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
     * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/wjkjssydclbvb/)
+* 2022-08-23 | Toronto, ON, CA | [Rust Toronto](https://www.meetup.com/rust-toronto/)
+    * [**WebAssembly plugins in Rust**](https://www.meetup.com/rust-toronto/events/287284601/)
 
 ### Oceania
 
 * 2022-07-28 | Brisbane, QLD, AU | [Rust Brisbane](https://www.meetup.com/rust-brisbane/)
     * [**July Meetup**](https://www.meetup.com/rust-brisbane/events/286889804/)
+* 2022-08-23 | Wellington, NZ + Virtual | [Rust Wellington](https://www.meetup.com/rust-wellington/)
```

Comment:
Seems like the date shifted, per the Meetup page
```suggestion
* 2022-08-24 | Wellington, NZ + Virtual | [Rust Wellington](https://www.meetup.com/rust-wellington/)
```

Reply:
Such a good find!!! Thank you!